vita3k/shader/src/usse_translator_entry.cpp

```diff
@@ -37,11 +37,11 @@ using USSEMatcher = shader::decoder::Matcher<Visitor, uint64_t>;
 template <typename V>
 boost::optional<const USSEMatcher<V> &> DecodeUSSE(uint64_t instruction) {
     static const std::vector<USSEMatcher<V>> table = {
-    // clang-format off
-    #define INST(fn, name, bitstring) shader::decoder::detail::detail<USSEMatcher<V>>::GetMatcher(fn, name, bitstring)
+#define INST(fn, name, bitstring) shader::decoder::detail::detail<USSEMatcher<V>>::GetMatcher(fn, name, bitstring)
+        // clang-format off
```

Comment:
What in the world happened here?

There's a comment disabling the formatter, and it had the guts to actually move it! Probably best if we unstage it.

Reply:
Set this before get problèm, it get all time change with run clang format